tutorials/7-cosmjs/5-create-custom.md

```diff
@@ -185,12 +286,29 @@ Each time `telescope transpile` has been ran, a config file `.telescope.json` wi
 You can modify the config file according to [telescope options](https://github.com/cosmology-tech/telescope#options), and then use the file by the option `--config`:
 
 
+<CodeGroup>
+<CodeGroupItem title="Local">
+
+```sh
+$ npx telescope transpile --protoDirs ../../proto-common --config .telescope.json
+```
+
+</CodeGroupItem>
+<CodeGroupItem title="Docker">
+
 ```sh
-# Telescope takes chain1 folder(from args) and chain2 folder(from config) as input,
-# and generate files in 'gen/src'(defined in the config file, will override outPath in args) folder using a config file.
-$ npx telescope transpile --protoDirs ../../proto/chain1 --config .telescope.json
+$ docker run --rm -it \
+    -v $(pwd):/myLib \
+    -w /myLib \
+    checkers_i \
+    npx telescope transpile --protoDirs ../../proto-common --config .telescope.json
 ```
 
+</CodeGroupItem>
+</CodeGroup>
+
+When running above command, telescope takes proto-common folder(from args) and proto folder(from config) as input, and generate files in 'gen/src'(defined in the config file, will override outPath in args) folder using a config file.
```

Comment:
```suggestion
When running the above command, telescope takes the `proto-common` folder (from arguments) and `proto` folder (from the configuration file) as inputs, and generates files in `gen/src` (defined as `outPath` in the configuration file, which overrides `outPath` if present in arguments) folder using a configuration file.
```

Still a bit confusing. Perhaps bullet points?

Reply:
Thanks! I tried to make it more clear!